test/success/derived-rule.prl

```diff
@@ -17,42 +17,42 @@ Rule MyRule : [
 // it replaces the current goal with the premises induced by the derived rule. The
 // hypothetical contexts are also unified together in the appropriate way (i.e. we 
 // do not require strict equality of contexts, but only compatibility).
-Thm MyCorollary : [ bool -> bool ] by [
+Thm MyCorollary : [ (-> bool bool) ] by [
```

Comment:
Oh, I was thinking that maybe the outermost parentheses can be omitted, but keeping them is indeed more s-expression-ish.